examples/f5_bigip/README.md

```diff
@@ -0,0 +1,353 @@
+        }
+    }
+    ```
+
+5. Create your pool members resources to manage membership in pools<sup>[3](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_pool_attachment)</sup>:
+
+    ```
+    resource "bigip_ltm_pool" "my_pool" {
+        name                   = "/${var.f5_partition}/pool_${var.test_site_name}"
+        load_balancing_mode    = "round-robin"
+        minimum_active_members = 1
+        monitors               = ["/Common/http"]
+    }
+
+    resource "bigip_ltm_pool_attachment" "my_pool_node" {
+        pool     = bigip_ltm_pool.my_pool.name
+        for_each = toset(var.f5_pool_members)
+        node     = each.key
+    }
+    ```
+
+6. Create you resource in order to create your virtual server to manage your F5 partition<sup>[4](https://registry.terraform.io/providers/F5Networks/bigip/latest/docs/resources/bigip_ltm_virtual_server)</sup>:
+
+    ```
+    resource "bigip_ltm_virtual_server" "my_virtual_server" {
+        name                       = "/${var.f5_partition}/vs_${var.test_site_name}"
+        description                = "Provisioned by Terraform"
+        destination                = var.f5_virtual_ip
+        port                       = var.f5_virtual_port
+        client_profiles            = [bigip_ltm_profile_client_ssl.my_profile.name]
+        source_address_translation = "automap"
+        pool                       = bigip_ltm_pool.my_pool.name
+    }
+    ```
+
+### Step 5: Apply your setup
+
+Finally, execute `terraform init`, ``terraform plan`` and ``terraform apply`` to apply your configuration changes. Then you should be able to log into your F5 partition in `192.168.x.x` using ``<your_f5_user>:<your_password>``.
+
+If done correctly, you should see an output similar to the following:
+
+[![asciicast](https://asciinema.org/a/fKmvGRMCGxSaLN6EodmedsRpg.svg)](https://asciinema.org/a/fKmvGRMCGxSaLN6EodmedsRpg)
+To tear down your F5 partition execute `terraform destroy`, then you should see an output like this:
+
+[![asciicast](https://asciinema.org/a/V7TthUjHaejww5miuUbAnmyQS.svg)](https://asciinema.org/a/V7TthUjHaejww5miuUbAnmyQS)
+
+## What's next
+<!-- should keep this section brief; if the answer is more than a small paragraph, I suggest that you link to another article/topic/website somewhere -->
+
+After you've successfully implemented this example, consider the following tips:
+
+- **What happens when certificates expire? How do they get renewed?** (BriefAnswerHere)
+
+- **How do certificates get validated?** (BriefAnswerHere)
+
+<!-- Depending on your MD language, you could format these as expandable text so users can click the bullet item to reveal your answers. -->
+
+## Helpful references 
+<!-- I think if you use this idea of providing links to third-party resources, that we should make it part of our template; otherwise, I would move these links one at a time up into your content where it makes the most sense and is in context -->
```

Comment:
Perhaps then we would go with "move these links to content" and remove this section, since it may not be able to be put in all examples as part of the template, because not all config options may have a direct link for the configuration (this if the provider is not registered in Terraform Registry).

Reply:
I agree. So I suggest you move them to where they make sense; might be easier to see and manage them if you put them directly below the paragraph they relate to and format them in their own paragraph with something like this: 

>For more information, see <a>Using jet fuel to speed up your BMW</a>.